common/swagger/swagger.yaml

```diff
@@ -141,9 +164,70 @@ definitions:
     type: array
     items:
       type: string
+
+  ContributionRequest:
+    type: object
+    required:
+      - requestShapleyValueType
+    properties:
+      requestShapleyValueType:
+        $ref: '#/definitions/ShapleyType'
+        description: >
+          The string to say what type of Shap values are needed.
+          `ORIGINAL` implies Shap values of original features are requested,
+          `TRANSFORMED` implies that Shap values of transformed features are requested.
+        type: string
+      fields:
+        description: >
+          An array holding the names of fields in the order of appearance in the `rows` property. The length of `fields`
+          has to match length of each row in `rows`. No duplicates are allowed.
+        type: array
+        items:
+          type: string
+      rows:
+        description: >
+          An array of rows consisting the actual input data for scoring, one scoring request per row.
+        type: array
+        items:
+          $ref: '#/definitions/Row'
+  ContributionResponse:
+    type: object
+    properties:
+      features:
+        description: >
+          An array holding the names of fields in the order of appearance in the rows of the  `contributions` property.
+        type: array
+        items:
+          type: string
+      contributionByOutputGroup:
+        description: >
+          An array of rows consisting of the shapley contributions output corresponding to an output group.
+        type: array
+        items:
+          $ref: '#/definitions/ContributionByOutputGroup'
+  ContributionByOutputGroup:
+    type: object
+    properties:
+      outputGroup:
+        description: >
+          Name of the output group.
+        type: string
```

Comment:
In the case of binomial/regression models, it looks like this `outputGroup` field is left out from the response.

Since the `outputGroup` is just the name of the model's output column, does it make sense to return these for even binomial/regression?

Although, 3rd party models don't necessarily have the notion of a column name for the output (unless the scorers of those models decide to populate this field with some generic "output", or something).

Whatever the decision is, I think we should adjust this swagger doc to document (even just in the description) what this field should always, or may not be guaranteed to, contain.

Reply:
Added additional documentation
```
description: >
          Name of the output group. It will be populated only for multinomial models of mojo2.
          Shapley values are not supported for third party models yet, hence this field will not be populated.
```